src/dat/view.cljs

```diff
@@ -755,8 +755,8 @@
                        (cljs-time/year dt)
                        (cljs-time/month dt)
                        (cljs-time/day dt)
-                       (input-time/time->hrs time-int)
-                       (input-time/time->mins time-int)
+                       (re-com.input-time/time->hrs time-int)
```

Comment:
No; either way. For things that aren't used too frequently I'm fine with more verbose if it ends up being clearer where the var is coming from inline. As you see fit.